minetest/minetest
# Batched rendering of particles

rebased and partially reworked #13834

compared to the original PR this has *no* spawn distance or global particle limit. this is better done in another PR if desired.

**This PR breaks transparency sorting of semi-transparent particles** (if it even worked before).

## To do

This PR is Ready for Review.

**note**: squash when merging. Make sure the co-authored-by line shows up for @x2048 and @Desour 

## How to test

1. try the code from here #13834
2. check that dig particles work
3. try the code from https://gist.github.com/sfan5/f3b8c427c5873474c947df20ff689e22
